src/axom/sidre/core/MFEMSidreDataCollection.cpp

```diff
@@ -953,18 +953,51 @@ void MFEMSidreDataCollection::Load(const std::string& path,
   }
 }
 
-void MFEMSidreDataCollection::LoadExternalData(const std::string& path)
+void MFEMSidreDataCollection::LoadExternalData(const std::string& filename,
+                                               const std::string& group_name)
 {
+  // Use the user-provided group name or the DataCollection's base group
+  Group* grp = m_bp_grp->getDataStore()->getRoot();
+  if(!group_name.empty())
+  {
+  #if defined(AXOM_USE_MPI) && defined(MFEM_USE_MPI)
+    if(m_comm != MPI_COMM_NULL)
+    {
+      SLIC_ERROR(
+        "Loading external data with a group name is not supported in "
+        "parallel.");
+    }
+  #endif
+
+    SLIC_ERROR_IF(!m_bp_grp->hasGroup(group_name),
+                  axom::fmt::format(
+                    "MFEMSidreDataCollection does not have a Sidre Group '{}'",
+                    group_name));
+    grp = m_bp_grp->getGroup(group_name);
+  }
+
+  // Use the user-provided file name or the DataCollection's file name
+  std::string path = name;
+  if(!filename.empty())
+  {
+    path = filename;
+  }
+  path = get_file_path(path);
+
   #if defined(AXOM_USE_MPI) && defined(MFEM_USE_MPI)
   if(m_comm != MPI_COMM_NULL)
   {
+    // The conduit abstraction appears to automatically handle the ".root"
```

Comment:
Would it be better to move the logic for the path suffix into the reader.loadExternalData() method?

Reply:
I am willing to put this in this PR or make an issue of it. I was just following the previous pattern and advice of @nselliott (i think)